src/app/directives/url/url-active.directive.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ChangeDetectorRef,
+  ContentChildren,
+  Directive,
+  ElementRef,
+  Input,
+  Optional,
+  QueryList,
+  Renderer2,
+} from "@angular/core";
+import {
+  IsActiveMatchOptions,
+  Router,
+  RouterLink,
+  RouterLinkActive,
+} from "@angular/router";
+import { UrlDirective } from "./url.directive";
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: "a[bawUrlActive]",
+})
+export class UrlActiveDirective extends RouterLinkActive {
```

Comment:
I think I'm going to need a summary table of these different directives

Reply:
Updated PR description to hopefully give you what you are looking for, not sure i've achieved that goal tho

Reply:
Add: example usage, input type, how qsps are used.

Also: why are there separate `*Active` directives?